lib/pure/collections/sequtils.nim

```diff
@@ -1067,9 +1068,30 @@ when isMainModule:
     doAssert mapLiterals(([1], ("abc"), 2), `$`, nested=true) == (["1"], "abc", "2")
 
   block: # mapIt with openArray
-    when declared(macros.symKind):
-      proc foo(x: openArray[int]): seq[int] = x.mapIt(it + 1)
-      doAssert foo([1,2,3]) == @[2,3,4]
+    proc foo(x: openArray[int]): seq[int] = x.mapIt(it + 1)
+    doAssert foo([1,2,3]) == @[2,3,4]
+
+  block: # mapIt with direct openArray
+    # see https://github.com/_render_node/MDExOlB1bGxSZXF1ZXN0MjA3MTQ3MTIw/pull_requests/timeline#issuecomment-411614711
+    doAssert openArray[int]([1,2]).mapIt(it) == @[1,2]
```

Comment:
A better test would be to create a `proc` that takes an `openarray`. Pass it a list and call mapIt on the argument.

Frankly, I'm surprised this works. ``openArray`` is a pseudo-type AFAIK.

Reply:
i already have a test for that, see "mapIt with openArray"